Clean-up/clean_lambda.py

```diff
@@ -0,0 +1,55 @@
+"A script to remove unsubscribed products from the database"
+
+from os import environ as ENV
+import logging
+import json
+
+from dotenv import load_dotenv
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection, cursor
+
+
+def get_connection() -> connection:
+    "Establishes a connection with the database"
+    return connect(
+        user=ENV["DB_USER"],
+        password=ENV["DB_PASSWORD"],
+        host=ENV["DB_HOST"],
+        port=ENV["DB_PORT"],
+        database=ENV["DB_NAME"]
+    )
```

Comment:
It would be better to pass through environment variables as a parameter:
```suggestion
def get_connection(config: dict) -> connection:
    "Establishes a connection with the database"
    return connect(
        user=config["DB_USER"],
        password=config["DB_PASSWORD"],
        host=config["DB_HOST"],
        port=config["DB_PORT"],
        database=config["DB_NAME"]
    )
```

Reply:
Done but change type hint to _Environ